pkg/morph/deploy/deploy.go

```diff
@@ -134,7 +134,9 @@ func Deploy(ctx context.Context, prm Prm) error {
 		return errors.New("local account does not belong to any Neo committee member")
 	}
 
-	monitor, err := newBlockchainMonitor(prm.Logger, prm.Blockchain)
+	chNewBlock := make(chan struct{}, 1)
```

Comment:
why `1`? what does it solve?

Reply:
non-blocking write with `select`

Reply:
IMO we either think that block handling is good enough and never have buffered channels or make its buffer bigger. if we are handling one block longer than the block lasts it is a possible deadlock (we saw and fixed it so many times).

`1` confuses me, so my question is what situation exactly you tried to solve here?

Reply:
we receive new block and try to send signal balance routine:
* if it's not busy, it receives signal from channel and makes balance actions
* if it's busy (currently making actions), we skip action for current block and will try again on future ones